packages/zoe/src/contracts/coveredCall-durable.js

```diff
@@ -0,0 +1,81 @@
+// @ts-check
+
+import { fit, M } from '@agoric/store';
+import '../../exported.js';
+
+import { vivifyKind, vivifySingleton } from '@agoric/vat-data';
+import { swapExact } from '../contractSupport/index.js';
+import { isAfterDeadlineExitRule } from '../typeGuards.js';
+
+const { details: X } = assert;
+
+const sellSeatExpiredMsg = 'The covered call option is expired.';
+
+/**
+ * @see comment on functionality in ./coveredCall.js.
+ *
+ * This variant has been made durable and upgradeable. The exerciser survives
+ * upgrade so the exerciseOption invitation will continue to work after upgrade.
+ *
+ * The version in
+ * zoe/test/swingsetTests/upgradeCoveredCall/coveredCall-durable-V3.js
```

Comment:
nit, that other one is the second durable. so I'd suggest `coveredCall-durable-V2.js` for it and `coveredCall-durable-V1.js` for this one.